lib/src/widgets/flutter_deck_footer.dart

```diff
@@ -66,16 +75,19 @@ class FlutterDeckFooter extends StatelessWidget {
         mainAxisAlignment: MainAxisAlignment.spaceBetween,
         crossAxisAlignment: CrossAxisAlignment.end,
         children: [
-          if (showSocialHandle)
+          if (widget case final Widget widget)
+            DefaultTextStyle(
+              style: FlutterDeckTheme.of(context).textTheme.bodySmall,
+              child: widget,
+            )
+          else if (showSocialHandle)
             Text(
               context.flutterDeck.speakerInfo?.socialHandle ?? '',
               style: theme.socialHandleTextStyle?.copyWith(
                 color: theme.socialHandleColor,
                 fontWeight: FontWeight.bold,
               ),
-            )
-          else
-            const SizedBox.shrink(),
+            ),
```

Comment:
The else case with `const SizedBox.shrink()` is still needed. Otherwise, when footer widget is not provided and  `showSocialHandle` is false, the slide number will be rendered on the left of the slide.

Reply:
You're absolutely right, I confused the `crossAxisAlignment: CrossAxisAlignment.end,` with the `mainAxisAlignment`... 🤦